### ISSUE TITLE
Make tip screen reader friendly

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#293505]
Behind an experiment, we have a tips UI that shows a message while chat is streaming in near the top of the chat response. This isn't accessible ATM, but should be. Let's discuss, but i'm thinking we can aria alert one time and add a command to focus it or alert if not present.

**Comment by meganrogge** [comment]
Key binding idea to toggle focus: `ctrl+shift+/`
